layouts/_default/contact.html

```diff
@@ -1,9 +1,8 @@
 {{ define "customCSS" }}
-    {{ $baseURL := .Site.BaseURL }}
-    <link rel="stylesheet" href="{{ $baseURL }}css/contact.css">
+    <link rel="stylesheet" href="/css/contact.css">
     <style>
         body {
-            background-image: url("{{ $baseURL }}images/bg-texture.png");
```

Comment:
Oh man I didnt realize this had to be done manually. Sorry if this took a lot of time